### PR TITLE
feat: prepare implementation tasks

### PR DESCRIPTION
**Pull request by hosekpeter**
Jira: [https://keboola.atlassian.net/browse/PSGO-382](url)

**Changes:**
- added structs for each commands flags
-----------

### CODE REVIEW

File: internal/pkg/service/cli/cmd/ci/flags.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package ci
+
+type WorkflowFlags struct {
+	CI           bool   `mapstructure:"ci" usage:"generate workflows"`
```
Comment (michaljurecko) [inline internal/pkg/service/cli/cmd/ci/flags.go:4]: `configkey:...` `configusage:...` ....

File: internal/pkg/service/cli/cmd/local/fix_paths.go [hunk 2]
```diff
@@ -39,7 +40,8 @@ func FixPathsCommand(p dependencies.Provider) *cobra.Command {
 		},
 	}
 
-	// Flags
-	cmd.Flags().Bool("dry-run", false, "print what needs to be done")
+	fixPathsFlag := FixPathsFlag{}
+	_ = cliconfig.GenerateFlags(fixPathsFlag, cmd.Flags())
+
```
Comment (michaljurecko) [inline internal/pkg/service/cli/cmd/local/fix_paths.go:45]:
```
cliconfig.MustGenerateFlags(FixPathsFlag{}, cmd.Flags())
```

Comment (michaljurecko) [reply internal/pkg/service/cli/cmd/local/fix_paths.go:45]:
```
cliconfig.MustGenerateFlags(FixPathsFlag{
   Foo: Bar
}, cmd.Flags())
```

Comment (michaljurecko) [reply internal/pkg/service/cli/cmd/local/fix_paths.go:45]: RunE:

```
flags := DefaultFixPathsFlag()
cliconfig.Bind(flags, ....)
```

Comment (michaljurecko) [reply internal/pkg/service/cli/cmd/local/fix_paths.go:45]: File structure?
```
type FooFlags...

func DefaultFooFlags

func FooCommand
```

Comment (hosekpeter) [reply internal/pkg/service/cli/cmd/local/fix_paths.go]: `flags := DefaultFixPathsFlag()
cliconfig.Bind(flags, ....)`

it will be in the next task

**Review by michaljurecko — APPROVED** [review]
LGTM

**Pull request merged**